load-balancing/elb-v2/common_functions.sh

```diff
@@ -374,15 +380,23 @@ wait_for_state() {
         return 1
     fi
 
-    msg "Checking $WAITER_ATTEMPTS times, every $waiter_interval seconds, for instance $instance_id to be in state $state_name"
+    # Check if a custom waiter_attempts was passed into the function
+    # and override the attemps if true
+    if [ -n ${var+5} ]; then
```

Comment:
I looked it up: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash . Is 5 a variable here or a constant?